### ISSUE TITLE
Missing keyboard navigation

### ISSUE THREAD

**Issue by ingatellent** [asivery/rm-appload#15]
Two issues:

1. Keyboard navigation is impossible in the appload lauch screen - it would be nice if the icons were selectable by the keyboard on the appload launch screen.

2. Appload (and Literm) in the sidebar cannot be opened using keyboard (they need to be added to the navigation list), right now using the keyboard (type folio) in the sidebar, the appload entry is just skipped. I could provide a fix for this second issue, but only for appload, as I have been unable to use REBUILD for the navigation list (I'm unsure why, might be a qmldiff-bug.).

**Comment by ingatellent** [comment]
Actually

`AFFECT [[4911547370760691430]]
    TRAVERSE [[3819512207256720568]]#[[233744436321411]]
        REBUILD [[15578034235503008618]]
            LOCATE BEFORE STREAM \~&7709552972437696890&~\ INSERT STREAM \openAppLoad,\
        END REBUILD
    END TRAVERSE
END AFFECT`

would solve 2.
